test/helpers/run.js

```diff
@@ -0,0 +1,20 @@
+const execa = require('execa');
+
+let project;
+
+function setProject(_project = undefined) {
+  project = _project;
+}
+
+function run(args, options = {}) {
+  options.reject = false;
+  options.cwd = options.cwd || project.path('.');
```

Comment:
Lets just do `options.cwd || process.cwd()`. Both the `cli-test.js` and the new file call `process.chdir(project.path())` in their setup (and reset in their teardown).

Reply:
OK that worked. Thanks.